Data Structures/Array Implementation/ArrayImplementation(Insert,Delete,Linear Search).cpp

```diff
@@ -0,0 +1,94 @@
+//A programme for basic Array Implmentation - Insertion, Deletion, Search (Linear).
+
+#include <stdio.h>
+
+int linearsearch(int value, int n, int array[]) {
+//Function which performs a linear search and returns the index.
+    int p = -1;
```

Comment:
Write clear names for the variables. **p** could be renamed to **position** which would make the code easier to read. Do this wherever possible.